### ISSUE TITLE
Command line flag does not change DBhost

### ISSUE THREAD

**Issue by trejas** [agenda/agenda-rest#14]
``agenda-rest -h 172.18.0.3 -d Agenda``

throws the following error:
```
MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (/usr/local/lib/node_modules/agenda-rest/node_modules/mongodb-core/lib/topologies/server.js:336:35)
    at Pool.emit (events.js:180:13)
    at Connection.<anonymous> (/usr/local/lib/node_modules/agenda-rest/node_modules/mongodb-core/lib/connection/pool.js:280:12)
    at Object.onceWrapper (events.js:272:13)
    at Connection.emit (events.js:180:13)
    at Socket.<anonymous> (/usr/local/lib/node_modules/agenda-rest/node_modules/mongodb-core/lib/connection/connection.js:189:49)
    at Object.onceWrapper (events.js:272:13)
    at Socket.emit (events.js:180:13)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at process._tickCallback (internal/process/next_tick.js:114:19)
```
Looks like it is still trying to connect to the localhost DB. Or am I doing something wrong?

**Comment by 2bigO** [comment]
Interesting, thanks for reporting @trejas 
Checking on this now. Meanwhile, could you please check whether `--dbhost` and `--dbname` work? So like this:

```shell
agenda-rest --dbhost 172.18.0.3 --dbname Agenda
```

#### UPDATE

That was not the cause, `seetings.js` would have duplicates in the transpile process. Fixed in 9ec7310

**Comment by 2bigO** [comment]
@trejas should be fixed in `0.7.3`. Appreciate if you could confirm 🎉 Feel free to reopen if the issue persisted.
